adhocracy4/ratings/static/ratings/react_ratings.jsx

```diff
@@ -23,8 +25,10 @@ var RatingBox = React.createClass({
   },
   handleRatingModify: function (number, id) {
     api.rating.change({
-      object_pk: this.props.objectId,
-      content_type: this.props.contentType,
+      urlReplaces: {
```

Comment:
They required knowledge already before. Previously they had to be specified in `data` too and were deleted afterwards. Now they still have to be in data but the values are grouped together.